kafkabp/config_test.go

```diff
@@ -108,4 +109,32 @@ func TestConsumerConfig(t *testing.T) {
 			t.Errorf("expected sarama rack id to be empty, got %q", sc.ClientID)
 		}
 	})
+
+	cfg.AutoCommitIntervalSeconds = 8
+	t.Run("auto-commit-interval", func(t *testing.T) {
+		sc, err := cfg.NewSaramaConfig()
+		if err != nil {
+			t.Errorf("expected no error, got %v", err)
+		}
+		if sc == nil {
+			t.Fatal("expected config to be non-nil, got nil")
+		}
+		if sc.Consumer.Offsets.AutoCommit.Interval != 8*time.Second {
+			t.Errorf("expected auto-commit interval to be %q, got %q", 8, sc.Consumer.Offsets.AutoCommit.Interval)
+		}
```

Comment:
```suggestion
		if got := sc.Consumer.Offsets.AutoCommit.Interval, want := 8*time.Second; got != want {
			t.Errorf("got auto-commit interval = %q, want %q", got, want)
		}
```

avoids some potentially subtle copy pasta errors for future developers where they update the conditional but not the error message